### PR TITLE
Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by ShadyShawkat**
In this project, I set up the mobile version of the portfolio using the following:

  1. Set up linters.
  2. Add HTML and CSS files.
  3. Add images of the project in a separate folder.
  4. Change the README.md file to properly describe the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      </nav>
+
+      <main>
+        <header>
+          <h1 class="header-title">Hey There.<br/> I'm Shady</h1>
+          <p class="primary-text">I’m a Software Developer</p>
+          <p class="describtion">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+
+          <div class="social-media-icons">
```
Comment (Ghiftee) [inline index.html:35]: - [ ] It is better to use a section tag for this container as it makes for better semantic HTML. Semantically correct HTML helps search engines, screen readers, and other user devices determine the significance and context of web content

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@200&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div>
+      <nav>
+          <span class="name">Shady Shawkat</span>
```
Comment (Ghiftee) [inline index.html:20]: - [ ] This should be a logo, good job aligning it correctly but please make it a link.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @ShadyShawkat 

Good job working on the project so far! 
✔️  You followed the design on Figma perfectly well
✔️ Your PR title and summary are descriptive
✔️  All your images work
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
 - [ ] One of the images on the page currently overlaps the text next to it, please fix it for a more professional project. You can find the overlapping bit below
Screenshot 2021-10-20 at 16 07 38
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 2 above)
Comment (ShadyShawkat) [reply index.html:20]: Done.

**Pull request merged**